test/functional/transactions.test.js

```diff
@@ -122,25 +124,12 @@ const SKIP_TESTS = [
 
 describe('Transactions Spec Legacy Tests', function () {
   const testContext = new TransactionsRunnerContext();
-  const suitesToRun = [{ name: 'spec tests', specPath: path.join('transactions', 'legacy') }];
-  // Note: convenient-api tests are skipped for serverless
-  if (!process.env.SERVERLESS) {
-    suitesToRun.push({
-      name: 'withTransaction spec tests',
-      specPath: path.join('transactions', 'convenient-api')
-    });
-  } else {
-    // FIXME(NODE-3550): these tests should pass on serverless but currently fail
-    SKIP_TESTS.push(
-      'abortTransaction only performs a single retry',
-      'abortTransaction does not retry after Interrupted',
-      'abortTransaction does not retry after WriteConcernError Interrupted',
-      'commitTransaction does not retry error without RetryableWriteError label',
-      'commitTransaction is not retried after UnsatisfiableWriteConcern error',
-      'commitTransaction fails after Interrupted'
-    );
-  }
-  suitesToRun.forEach(suiteSpec => {
+  const suitesToRun = [
+    { name: 'spec tests', specPath: path.join('transactions', 'legacy') },
+    { name: 'withTransaction spec tests', specPath: path.join('transactions', 'convenient-api') }
```

Comment:
I'm confused, were the convenient-api tests just not ran at all before?

Reply:
Is it because of this: 

> Note: the killAllSessions command is not supported on Serverless, so the transactions tests may hang if an individual test leaves a transaction open when it finishes (CLOUDP-84298)

I mean it specifically tells us not to test them, so I would follow up with the drivers group to make sure we understand the implications; I don't want to run into a situation where future changes cause things to fail and then we have to spend time investigating why it's happening; at a minimum, if we do keep them, I would make sure to put a comment above explicitly saying that "these are not required to run for serverless, we are just choosing to because they work"

Reply:
Well grepping the logs for "withTransaction" I'm seeing nothing so somehow they are still skipped 🤔